raiden/transfer/mediated_transfer/target.py

```diff
@@ -232,30 +232,32 @@ def handle_block(
     """
     transfer = target_state.transfer
     events = list()
+    lock = transfer.lock
```

Comment:
Just checking. A `transfer` here is a `LockedTransferUnsignedState` so it's `lock` can't be None. Right?

Reply:
yep